### ISSUE TITLE
Suppress row group label and double horizontal rule

### ISSUE THREAD

**Issue by vincentarelbundock** [rstudio/gt#140]
It would be nice if users could suppress group labels when using the `tab_row_group` or the `group_by` strategies. Currently, we can use the `tab_row_group` function and leave the `group` argument empty, but this produces a double horizontal rule.

A workaround would be to allow users to insert horizontal rules where they want (is there such a function now?).

```r
library(gt)
library(dplyr)

dat <- data.frame('a' = 1:4,
                  'b' = c('groupA', 'groupA', 'groupB', 'groupB'))

# idiomatic grouping strategy prints group labels by default
dat %>%
  dplyr::group_by(b) %>%
  gt()
```

![screen shot 2019-01-21 at 13 56 31](https://user-images.githubusercontent.com/987057/51493844-97bac880-1d84-11e9-849f-dcacb65be13e.png)

```r
# hack to suppress group labels. Unfortunately, this keeps a double horizontal rule.
dat %>%
  select(-b) %>%
  gt() %>%
  tab_row_group(group = "", rows = 1:2)
```
![screen shot 2019-01-21 at 13 56 39](https://user-images.githubusercontent.com/987057/51493841-95f10500-1d84-11e9-97f6-35f1577576c5.png)

**Comment by rich-iannone** [comment]
I agree with this, in that the row group label is not always useful and also that the double line (or perhaps any line) should be optional. Thanks!

**Comment by geneh0** [comment]
Stumbled upon this via Google and wanted to share my workaround to avoid the double horizontal rule. 

```
tab_style(
    style = cell_borders(
      sides = c("bottom"),
      color = "white",
      weight = px(0)
    ),
    location = cells_row_groups()
  )```
